File: codeChef/salary.py
def salary(w):
    count = sum(w) - (len(w)*min(w))
    return count   


if __name__ == "__main__":
    test_cases = int(input())
    ans = list()
    for i in range(0, test_cases):
        N = int(input())
        w = list(map(int, input().split()))
        ans.append(str(salary(w)))

    print("\n".join(ans))

File: codeChef/CPP files/ARRAY/salary.cpp
#include<iostream>
int main(){
    int T;
    std:: cin>>T;
    while(T--){
        int N;
        int sum=0, min=1000000; 
        int salary;
        std::cin>>N;
        for(int i=0; i<N; i++){
            std::cin>>salary;
            sum += salary;
            min = std::min(min, salary);
        }
        std::cout<<(sum - (N*min))<<std::endl;
    }
}